<launch>

	<group ns="camera">

	<group ns="front">
		<node name="image_proc" pkg="image_proc" type="image_proc"/>
	</group>

	<group ns="bottom">
		<node name="image_proc" pkg="image_proc" type="image_proc"/>
	</group>

	<node name="front" pkg="avt_vimba_camera" type="mono_camera_node" output="screen">
		<param name="guid" value="02-2141A-17143"/>
		<param name="ip_address" value=""/>
		<param name="camera_info_url" value="file://$(find avt_vimba_camera)/calibrations/calibration_50-0503343290.yaml"/>
		<param name="frame_id" value="left_optical"/>
		<param name="trig_timestamp_topic" value=""/>
		<param name="show_debug_prints" value="true"/>
		<param name="trigger_source" value="FixedRate"/>
		<param name="acquisition_mode" value="Continuous"/>
		<param name="acquisition_rate" value="3"/>
		<param name="pixel_format" value="BayerRG8"/>
		<param name="exposure" value="30000"/>
		<param name="gain" value="0"/>
		<param name="exposure_auto" value="Off"/>
		<param name="gain_auto" value="Continuous"/>
		<param name="whitebalance_auto" value="Continuous"/>
		<param name="binning_x" value="1"/>
		<param name="binning_y" value="1"/>
		<param name="decimation_x" value="1"/>
		<param name="decimation_y" value="1"/>
		<param name="x_offset" value="0"/>
		<param name="y_offset" value="0"/>
		<param name="width" value="1360"/>
		<param name="height" value="1024"/>
		<param name="stream_bytes_per_second" value="5000000"/>
	</node>

	<node name="bottom" pkg="avt_vimba_camera" type="mono_camera_node" output="screen">
		<param name="guid" value="02-2141A-17142"/>
		<param name="ip_address" value=""/>
		<param name="camera_info_url" value="file://$(find avt_vimba_camera)/calibrations/calibration_50-0503343290.yaml"/>
		<param name="frame_id" value="left_optical"/>
		<param name="trig_timestamp_topic" value=""/>
		<param name="show_debug_prints" value="true"/>
		<param name="trigger_source" value="FixedRate"/>
		<param name="acquisition_mode" value="Continuous"/>
		<param name="acquisition_rate" value="3"/>
		<param name="pixel_format" value="BayerRG8"/>
		<param name="exposure" value="30000"/>
		<param name="gain" value="0"/>
		<param name="exposure_auto" value="Off"/>
		<param name="gain_auto" value="Continuous"/>
		<param name="whitebalance_auto" value="Continuous"/>
		<param name="binning_x" value="1"/>
		<param name="binning_y" value="1"/>
		<param name="decimation_x" value="1"/>
		<param name="decimation_y" value="1"/>
		<param name="x_offset" value="0"/>
		<param name="y_offset" value="0"/>
		<param name="width" value="1360"/>
		<param name="height" value="1024"/>
		<param name="stream_bytes_per_second" value="5000000"/>
	</node>

	</group>
	
</launch>
